test/weets/create_weet.test.js

```diff
@@ -0,0 +1,69 @@
+const request = require('supertest');
+const app = require('../../app');
+const { userData, userDataAuthorized } = require('../users/data/users');
+
+describe('POST /weets user register weet into database', () => {
+  let status = null;
+  let weet = null;
+  let token = null;
+  beforeAll(async () => {
+    await request(app)
```

Comment:
In this kind of test there is no need to singup and singin, you can mock that in order to only test the developed functionality, otherwise the test is prompt to unrelated errors (tests would be coupled)

Reply:
Since it is not strictly unit testing, it is ok :+1: